src/content/docs/new-relic-one/use-new-relic-one/ui-data/maps-in-context.mdx

```diff
@@ -1,19 +1,21 @@
 ---
-title: Maps in context help you understand entity relationships
-metaDescription: Maps in context are available throughout the New Relic One UI to show how your entities relate to one another.
+title: Maps in context display entity relationships
+metaDescription: Maps in context are available in several New Relic UI locations to show how entities relate to one another.
 ---
 
-Maps show how your software entities are connected and deployed, using New Relic's auto-discovered and user-defined relationships for topology.
+Some New Relic features, like distributed tracing and Applied Intelligence, display topological maps that show relationships between apps, hosts, services, and other [entities](/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic) to aid in diagnosing and correcting issues. 
```

Comment:
If `distributed tracing` is a product or feature, then it should also be capitalized as `Applied Intelligence` is. Ignore this comment if I'm understanding distributed tracing incorrectly.

Reply:
we actually don't capitalize all features, only a few specifically branded things. It's a pretty short list, but includes things like Applied Intelligence, Incident Intelligence, Telemetry Data Platform, Full Stack Observability, a few others. In general, features that refer to just a non-branded functionality are not capitalized; another way to put this is: distributed tracing is just an industry term, not a New Relic branded term.